conf/ocsci/skip_blockpools.yaml

```diff
@@ -0,0 +1,3 @@
+COMPONENTS:
+  enable_blockpools: False
```

Comment:
If you are changing the filename to disable_blockpools.yaml, then it would be good have parameter disable_blockpools: True